ga4gh/datamodel/reads.py

```diff
@@ -110,22 +109,22 @@ def __init__(self, id_):
         self._readGroups.append(readGroup)
 
 
-class HtslibReadGroupSet(datamodel.PysamSanitizer, AbstractReadGroupSet):
+class HtslibReadGroupSet(AbstractReadGroupSet, datamodel.PysamDatamodelMixin):
```

Comment:
While this shouldn't matter for genuine mixins (which have no business redefining symbols from the "true" base class), I support @dcolligan's reasoning. Good blog post about this [here](http://www.ianlewis.org/en/mixins-and-python).